### PR TITLE
Balancer: liquidity views

### PR DESCRIPTION
**Pull request by mendesfabio**
I've checked that:

* [x] the query produces the intended results
* [x] the folder name matches the schema name
* [x] the schema name exists in Dune
* [x] views are prefixed with `view_`, functions with `fn_`.
* [x] the filename matches the defined view, table or function and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

### CODE REVIEW

File: ethereum/balancer_v2/view_liquidity.sql [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+            LEFT JOIN erc20.tokens t ON t.contract_address = c.token
+            LEFT JOIN pool_labels p ON p.pool_id = SUBSTRING(b.pool :: text, 0, 43) :: bytea
+    )
+    SELECT
+        *
+    FROM
+        balancer_liquidity
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS balancer_v2_view_liquidity_idx ON balancer_v2.view_liquidity (DAY, pool_id, token_address);
```
Comment (antonio-mendes) [inline ethereum/balancer_v2/view_liquidity.sql:325]: I'm getting a `psql:view_liquidity.sql:325: ERROR:  column "pool_id" does not exist` error here. Could you double check and commit a fix? I ran the query by itself and saw the `pool_id` field was indeed missing.

**Review by antonio-mendes — CHANGES_REQUESTED** [review]
Oi @mendesfabio ,

First of all let me say you have a beautiful last name 👀 

Secondly I dropped you a comment on an error I found with `balancer_v2.view_liquidity`.

And finally, I pushed a commit with a few small changes:
- explicitly declaring schema names
- small fix that was causing me problems when inserting to cron

Let me know if I can help you further, and I'll keep an eye on this PR.

Cheers!

**Comment by mendesfabio** [comment]
hey @antonio-mendes 😂 nice to meet you!

I just fixed the error you found, thanks! Let me know if you find anything else.

**Comment by antonio-mendes** [comment]
Thank you @mendesfabio!  PR merged and changes deployed, feel free to reach out if you notice anything weird

**Pull request merged**